app/models/transaction.rb

```diff
@@ -2,4 +2,5 @@ class Transaction < ApplicationRecord
   enum result: [:failure, :success]
 
   belongs_to :invoice
+
```

Comment:
This is probably an eternal war between us where I remove the empty line at the end and then you put it back in. You win this round...

Reply:
Pry it from my cold dead hands!